JASP-Engine/JASP/R/LSbinomialtesting.R

```diff
@@ -849,12 +854,27 @@ LSbinomialtesting   <- function(jaspResults, dataset, options, state = NULL){
       tablePredictions$addColumns(0:options[["predictionN"]])
     }
     
-    if (options[["plotsPredictionPostType"]] %in% c("joint", "conditional")){
-      for(i in 1:length(options[["priors"]])){
-        tablePredictions$addColumnInfo(name = paste0("hyp_", i), title = gettextf("P(Successes|%s)", options[["priors"]][[i]]$name), type = "number")
-      }
-    } else if (options[["plotsPredictionPostType"]] == "marginal")
-      tablePredictions$addColumnInfo(name = "marginal", title = gettextf("P(Successes)"), type = "number")
+    
+    if (ready[2]){
+      if (options[["plotsPredictionPostType"]] %in% c("joint", "conditional")){
+        for(i in 1:length(options[["priors"]])){
```

Comment:
Would
```suggestion
        for(i in seq_along(options[["priors"]])){
```
be perhaps better?

Reply:
yeah, probably